CMakeLists.txt

```diff
@@ -201,14 +201,19 @@ IF(WANT_LV2)
 	IF(PKG_CONFIG_FOUND)
 		PKG_CHECK_MODULES(LV2 lv2)
 		PKG_CHECK_MODULES(LILV lilv-0)
+	ENDIF()
+	IF(NOT LV2_FOUND AND NOT LILV_FOUND)
+		FIND_PACKAGE(LV2 CONFIG)
+		FIND_PACKAGE(LILV CONFIG)
 		IF(LV2_FOUND AND LILV_FOUND)
-			SET(LMMS_HAVE_LV2 TRUE)
-			SET(STATUS_LV2 "OK")
-		ELSE()
-			SET(STATUS_LV2 "not found, install it or set PKG_CONFIG_PATH appropriately")
+			SET(LILV_LIBRARIES "lilv::lilv")
 		ENDIF()
+	ENDIF()
+	IF(LV2_FOUND AND LILV_FOUND)
+		SET(LMMS_HAVE_LV2 TRUE)
+		SET(STATUS_LV2 "OK")
 	ELSE()
-		SET(STATUS_LV2 "not found, requires pkg-config")
+		SET(STATUS_LV2 "not found, install it or set PKG_CONFIG_PATH appropriately")
```

Comment:
I didn't really think about this - I just moved the message down from above. I would argue that "set `PKG_CONFIG_PATH` appropriately" implies that pkg-config needs to be installed - it can't be set appropriately if there's no pkg-config to point it at. I'm not completely familiar with how everything is acquired and found on other platforms, so I don't know what would be better to put here, but I'm open to suggestions.

Reply:
The "set `PKG_CONFIG_PATH` appropriately" argument makes sense. About the `FIND_PACKAGE(LV2 CONFIG)`, I don't know what else could lead to failure here (probably it's covered by "install it"?).

I think this comment is not a blocker, we can still change the message if users complain about not understanding the error.

Reply:
If the CMake config file isn't found, and vcpkg is being used, then either:
* The relevant ports aren't installed; this is already mentioned in the message,
* The vcpkg toolchain file isn't specified correctly, in which case there will be a bunch of other errors too, or
* There's a bug in vcpkg or in one of the ports, in which case they should come and speak to us anyway.

I'm going to resolve this conversation; feel free to unresolve it if there's something else that needs to be brought up.

Reply:
This PR doesn't change the error message - this one I moved down from earlier in the file, and the "requires pkg-config" one I removed since it's no longer true. I'll leave this up to @JohannesLorenz, in the case that we do want to fix it here.

Reply:
It's correct, the PR diff just looks like the error message was changed. It's from #4899 . Though, technically, we could change it here.

Imagine the user has the Lv2 pkg-config path installed outside of the systems pkg-config paths (local install) and gets "please install Lv2". I wonder if that makes clear that the message means "Yes, Lv2 may be installed, but I did not find it, give me the path where I need to search". What do you think, is it obvious by saying "please install Lv2"?

Reply:
@JohannesLorenz oh, so the fear is that a particular system may not have it provided and `PKG_CONFIG_PATH` needs to be added?  That makes sense, especially for packages that may not be available by the package managers.  I tested on 18.04 and Homebrew, both which provide this file.  16.04's package listing seems to offer it as well.  I think `PKG_CONFIG_PATH` is more of an edge-case reminder for things that the package manager doesn't provide (e.g. Carla) but even the edge-cases we've managed to workaround without custom PKG_CONFIG_PATHs.  I think it's safe to remove, or perhaps [added to the build summary notes](https://github.com/LMMS/lmms/blob/9d104b2205f63bd4425905e1261ffd63f57baba9/CMakeLists.txt#L761-L765).

Reply:
@tresf Agreed, it's a general thing that also affects other libs (like Carla), so I'm OK with removing it. @DomClark can you please change the error message?

Reply:
I'm also fine reverting this and I can fix the wording in a separate commit since `suil` should also be fixed.

Reply:
Revert? It's not introduced in this PR.

Separate commit looks cleaner to me, too, as this is not part of supporting Lv2 on MSVC.

Reply:
> Revert? It's not introduced in this PR.

Yes, sorry the diff keeps throwing me off.  @DomClark's stated that at least twice.  Marking as resolved. :)